src/flamegraph.js

```diff
@@ -437,12 +437,14 @@ export default function () {
             g.exit()
                 .remove()
 
-            g.on('mouseover', function (d) {
-                if (tooltip) tooltip.show(d, this)
-                detailsHandler(labelHandler(d))
-            }).on('mouseout', function () {
+            g.on('mouseout', function () {
                 if (tooltip) tooltip.hide()
                 detailsHandler(null)
+            }).on('mousemove', function (d) {
+                const chartBoundingClientRect = svg._groups[0][0].parentNode.getBoundingClientRect()
+                const chartRightBoundary = chartBoundingClientRect.x + chartBoundingClientRect.width
+                if (tooltip) tooltip.show(d, chartRightBoundary, this)
```

Comment:
@spiermar What is the use of passing the argument `this` while calling tooltip.show ?

Reply:
Compatibility with d3-tip, which I believe used it to calculate position.